CHANGES.rst

```diff
@@ -322,6 +322,13 @@ astropy.visualization
 astropy.wcs
 ^^^^^^^^^^^
 
+- Do not attempt to delete repeated distortion keywords multiple times when
```

Comment:
I assume this isn't supposed to be here?

Reply:
I actually rebased my branch, and it merged this change as well. Any suggestions to remove it using git, or shall I remove it manually?

Reply:
@yashrsharma44 - the rebase should never bring in a merge commit. I would suggest to rebase again. When you're done you need to force push your branch back to the remote rather that pull/merge.

Reply:
Actually, first I updated my `master` branch using `git pull upstream master` `upstream` being my remote to astropy repo. Then I performed `git rebase none-fix`, so it included the merge commit in the rebase. 
Now I have reset the merge, and tried doing rebase again, but it still catches those extra lines. May be I need to see which commit catches the extra lines. Thanks for the insight of never having a merge commit in a rebase, I will keep that in my mind.

Reply:
Now its correct, and I am getting a conflict error.

Reply:
@bsipocz Can you help me with the conflicting files?

Reply:
```
git fetch upstream master
git rebase upstream/master
```

Follow screen instructions to resolve conflict in editor.

```
git add CHANGES.rst
git rebase --continue
```

Reply:
Thanks @pllim for the suggestions, it worked!